packages/terraform-modules-public/gcp/celo-infra/modules/proxy/startup.sh

```diff
@@ -416,6 +413,7 @@ ExecStart=/usr/bin/docker run \\
       --proxy.proxy \\
       --proxy.proxiedvalidatoraddress ${validator_account_address} \\
       --proxy.internalendpoint :30503 \\
+      --light.serve 0 \\
```

Comment:
curious why this?

Reply:
Consensus was that validator proxies shouldn't be serving light clients.  See https://discord.com/channels/600834479145353243/697229909545713775/788845333052260405